### PR TITLE
On-prem prereqs updates

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Clean up 23.3 and 23.4 on-prem prereqs pages.

### CODE REVIEW

File: platform_versioned_docs/version-22.4.0/_todo.mdx [hunk 1]
```diff
@@ -61,3 +61,14 @@ AZURE Batch - SSL problem as per https://git.seqera.io/eagle/nf-support/issues/1
 https://github.com/nf-core/configs/blob/master/conf/google.config as per this ticket: https://git.seqera.io/pluto/nf-support/issues/6
 
 TO DO: As per Ben, document profile injection behaviour into Tower docs (e.g. nf-core automagically injecting google profile if executing on GLS)
+
+Azure prereqs notes dump:
```
Comment (jason-seqera) [inline platform_versioned_docs/version-22.4.0/_todo.mdx:65]: Should we keep this file after this PR is merged?

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx [hunk 2]
```diff
@@ -5,28 +5,30 @@ date: "12 Apr 2023"
...
 
-Ensure that you satisfy the requirements for your installation. Refer to this list of prerequisites:
+You must satisfy the requirements for your installation:
 
 - **SMTP server**: An SMTP server is required to send emails from your Seqera instance.
 
   If you don't have your own mail server, you can use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
 
-- **MySQL database**: An external database (i.e., external to your Docker Compose or Kubernetes deployment) is highly recommended for production deployments. If you don't have your own database service, you can use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
+- **MySQL database**: An database external to your Docker Compose or Kubernetes environment is highly recommended for production deployments.
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:22]: Is it _an_ or _a_ database?

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx [hunk 3]
```diff
@@ -36,20 +38,20 @@ container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](htt
...
    ```bash
    docker login -u '/\<USERNAME\>/' -p '/\PASSWORD\>/' cr.seqera.io
    ```
 
-3. Pull the Seqera container images with the following commands:
+3. Pull the Seqera container images:
 
    ```bash
-   docker pull cr.seqera.io/private/nf-tower-enterprise/backend:v23.3.0
+   docker pull cr.seqera.io/private/nf-tower-enterprise/backend:v23.4.3
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:50]: We can theoretically start using MDX variables for this now.

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx [hunk 4]
```diff
@@ -36,20 +38,20 @@ container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](htt
...
 
-   docker pull cr.seqera.io/private/nf-tower-enterprise/frontend:v23.3.0
+   docker pull cr.seqera.io/private/nf-tower-enterprise/frontend:v23.4.3
    ```
 
-:::caution
-`cr.seqera.io` is the default Seqera container image registry from version 22.4. Use of the AWS, Azure, and Google Cloud image registries in existing installations is still supported but is deprecated for **new installations** from June 2023.
-:::
+## Next steps
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:55]:
```suggestion
## Next steps
```

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx [hunk 5]
```diff
@@ -36,20 +38,20 @@ container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](htt
...
-   docker pull cr.seqera.io/private/nf-tower-enterprise/frontend:v23.3.0
+   docker pull cr.seqera.io/private/nf-tower-enterprise/frontend:v23.4.3
    ```
 
-:::caution
-`cr.seqera.io` is the default Seqera container image registry from version 22.4. Use of the AWS, Azure, and Google Cloud image registries in existing installations is still supported but is deprecated for **new installations** from June 2023.
-:::
+## Next steps 
+
+See [Configuration](../configuration/overview.mdx).
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:57]:
```suggestion
See [Configuration](../configuration/overview.mdx).
```

File: platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx [hunk 6]
```diff
@@ -36,20 +38,20 @@ container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](htt
...
    ```bash
    docker pull cr.seqera.io/private/nf-tower-enterprise/backend:v23.4.3
 
    docker pull cr.seqera.io/private/nf-tower-enterprise/frontend:v23.4.3
    ```
 
-:::caution
-`cr.seqera.io` is the default Seqera container image registry from version 22.4. Use of the AWS, Azure, and Google Cloud image registries in existing installations is still supported but is deprecated for **new installations** from June 2023.
-:::
+## Next steps
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx:55]:
```suggestion
## Next steps
```

File: platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx [hunk 7]
```diff
@@ -36,20 +38,20 @@ container registry [`cr.seqera.io`](https://cr.seqera.io). Contact [support](htt
...
 
    docker pull cr.seqera.io/private/nf-tower-enterprise/frontend:v23.4.3
    ```
 
-:::caution
-`cr.seqera.io` is the default Seqera container image registry from version 22.4. Use of the AWS, Azure, and Google Cloud image registries in existing installations is still supported but is deprecated for **new installations** from June 2023.
-:::
+## Next steps 
+
+See [Configuration](../configuration/overview.mdx).
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx:57]:
```suggestion
See [Configuration](../configuration/overview.mdx).
```

**Review by jason-seqera — COMMENTED** [review]
@llewellyn-sl, left a few minor comments.

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx (hunk 2 above)
Comment (justinegeffen) [inline platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:22]:
```suggestion
- **MySQL database**: A database external to your Docker Compose or Kubernetes environment is highly recommended for production deployments.
```

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx [hunk 8]
```diff
@@ -5,28 +5,30 @@ date: "12 Apr 2023"
...
+- **MySQL database**: An database external to your Docker Compose or Kubernetes environment is highly recommended for production deployments. 
 
-  If you decide to use an external database, you must create a MySQL user and database manually. See [Configuration](../configuration/overview.mdx#seqera-and-redis-databases) for more details.
+  If you don't have your own database service, use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
+
+  To use an external database, you must create a MySQL user and database manually. See [Configuration](../configuration/overview.mdx#seqera-and-redis-databases) for more details.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:26]:
```suggestion
  When you use an external database, you must create a MySQL user and database manually. See [Configuration](../configuration/overview.mdx#seqera-and-redis-databases) for more details.
```

File: platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx [hunk 9]
```diff
@@ -5,28 +5,30 @@ date: "12 Apr 2023"
...
+- **MySQL database**: An database external to your Docker Compose or Kubernetes environment is highly recommended for production deployments. 
 
-  If you decide to use an external database, you must create a MySQL user and database manually. See [Configuration](../configuration/overview.mdx#seqera-and-redis-databases) for more details.
+  If you don't have your own database service, use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
+
+  To use an external database, you must create a MySQL user and database manually. See [Configuration](../configuration/overview.mdx#seqera-and-redis-databases) for more details.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx:26]:
```suggestion
  When you use an external database, you must create a MySQL user and database manually. See [Configuration](../configuration/overview.mdx#seqera-and-redis-databases) for more details.
```

File: platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx [hunk 10]
```diff
@@ -5,28 +5,30 @@ date: "12 Apr 2023"
...
 
-Ensure that you satisfy the requirements for your installation. Refer to this list of prerequisites:
+You must satisfy the requirements for your installation:
 
 - **SMTP server**: An SMTP server is required to send emails from your Seqera instance.
 
   If you don't have your own mail server, you can use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
 
-- **MySQL database**: An external database (i.e., external to your Docker Compose or Kubernetes deployment) is highly recommended for production deployments. If you don't have your own database service, you can use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
+- **MySQL database**: An database external to your Docker Compose or Kubernetes environment is highly recommended for production deployments.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-23.4.0/enterprise/prerequisites/on-prem.mdx:22]:
```suggestion
- **MySQL database**: A database external to your Docker Compose or Kubernetes environment is highly recommended for production deployments. 
```

**Review by justinegeffen — APPROVED** [review]
I've added a few minor comments but otherwise this looks good!

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx (hunk 2 above)
Comment (llewellyn-sl) [reply platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:22]: Lol. Thanks. Feel free to suggest fixes or just commit directly in such silly typo cases :)

File: platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx (hunk 3 above)
Comment (llewellyn-sl) [reply platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx:50]: Sweet! Do we store all the global variables in a single file, or declare them at the start or end of individual MDX pages?

**Review by ShahzebMahmood — COMMENTED** [review]
LGTM

**Pull request merged**